components/notability/commons/notability_checker.lua

```diff
@@ -18,6 +18,8 @@ local _lang = mw.language.new('en')
 local _NOW = os.time()
 local _SECONDS_IN_YEAR = 365.2425 * 86400
 
+NotabilityChecker._EXTRA_LOGGING = true
```

Comment:
should the default maybe be `false`?

Reply:
Current production version has the logging active. So I set it to `true` as the default value

Reply:
Should be `true`, yes